test/server/compiler-test.js

```diff
@@ -140,6 +173,39 @@ describe('Compiler', function () {
 
 
     describe('TypeScript', function () {
+        it('Should compile test defined in separate module if option is enabled', function () {
```

Comment:
Why does we need addtional tests written with TypeScript and CoffeeScript?

Reply:
We have several compilers and every on them has it's own set of similar tests. It's true that I modified base behavior which affects all of the compilers at the same time, but I think it'd better to have full set of tests